wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/StreamFormat.java

```diff
@@ -32,6 +32,7 @@ public enum StreamFormat {
     Z16H(28),
     FG(29),
     Y411(30);
+    Y16I(31);
```

Comment:
StreamFormat.java:35: error: invalid method declaration; return type required
14:07:02      Y16I(31);
Please change previous line `;` to `,`

Reply:
Done